packages/ra-ui-materialui/src/button/DeleteButton.tsx

```diff
@@ -11,7 +11,7 @@ import DeleteWithConfirmButton from './DeleteWithConfirmButton';
  * Button used to delete a single record. Added by default by the <Toolbar> of edit and show views.
  *
  * @typedef {Object} Props The props you can use (other props are injected if you used it in the <Toolbar>)
- * @param {Prop} props
+ * @param {Props} props
```

Comment:
If you edit this one, you'd better move it out of the `typedef` definition - I don't think `@param` is valid under `@typedef`

Reply:
Done!